src/presentation/http/router/editorTools.ts

```diff
@@ -18,11 +31,18 @@ interface EditorToolsRouterOptions {
  * @param opts - empty options
  * @param done - callback
  */
-const EditorToolsRouter: FastifyPluginCallback<EditorToolsRouterOptions> = (fastify, opts, done) => {
+const EditorToolsRouter: FastifyPluginCallback<EditorToolsRouterOptions> = async (fastify, opts, done) => {
   /**
    * Manage editor tools data
    */
-  const { editorToolsService } = opts;
+  const { editorToolsService, fileUploaderService } = opts;
+
+  await fastify.register(fastifyMultipart, {
```

Comment:
I think the `fastify.registed` should be stored here
[notes.api/src/presentation/http/http-api.ts](https://github.com/codex-team/notes.api/blob/main/src/presentation/http/http-api.ts)